source/js/create-recipe.js

```diff
@@ -256,11 +213,15 @@
     recipe["steps"] = steps;
 
     if (allValid) {
-      const id = parseInt(recipes.currID, 10);
-      recipes.currID = id + 1;
+      // const id = parseInt(recipes.currID, 10);
+      console.log(recipes["currID"]);
+      recipes[recipes["currID"]] = recipe;
+      recipes["currID"] += 1;
 
-      recipes[id] = recipe;
       localStorage.setItem("recipes", JSON.stringify(recipes));
+      window.alert("successfully created a recipe!");
+      let currUrl = location.toString().replace("create-recipe.html", "cook-book.html");
```

Comment:
Codacy has a fix for the issue: ['currUrl' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/cse110-fa21-group7/cse110-fa21-group7/pullRequest?prid=8446960)
```suggestion
      const currUrl = location.toString().replace("create-recipe.html", "cook-book.html");
```